mmcv/ops/points_sampler.py

```diff
@@ -92,11 +92,12 @@ def __init__(self,
     def forward(self, points_xyz, features):
         """
         Args:
-            points_xyz (Tensor): (B, N, 3) xyz coordinates of the features.
-            features (Tensor): (B, C, N) Descriptors of the features.
+            points_xyz (torch.Tensor): (B, N, 3) xyz coordinates of the
+                features.
+            features (torch.Tensor): (B, C, N) Descriptors of the features.
```

Comment:
points_xyz (torch.Tensor): (B, N, 3) xyz coordinates of the points.
 features (torch.Tensor): (B, C, N) features of the points.